lib/ui/window.dart

```diff
@@ -426,19 +426,39 @@ class Window {
     _onMetricsChangedZone = Zone.current;
   }
 
-  /// The system-reported locale.
+  /// The system-reported default locale of the device.
   ///
   /// This establishes the language and formatting conventions that application
   /// should, if possible, use to render their user interface.
   ///
+  /// This is the first locale selected by the user and should be the user's
```

Comment:
Nit: "should be the user's primary locale" begs some questions:
- What's the primary locale exactly?
- When would it not be the primary locale?

Including a link to a public document that covered this kind of thing would be helpful.